chain-abstraction/sol-implementation/assertion_chain.go

```diff
@@ -204,7 +204,7 @@ func NewAssertionChain(
 		confirmedChallengesByParentAssertionHash: threadsafe.NewLruSet(1000, threadsafe.LruSetWithMetric[protocol.AssertionHash]("confirmedChallengesByParentAssertionHash")),
 		averageTimeForBlockCreation:              time.Second * 12,
 		transactor:                               transactor,
-		rpcHeadBlockNumber:                       rpc.FinalizedBlockNumber,
+		rpcHeadBlockNumber:                       rpc.LatestBlockNumber,
```

Comment:
Expects the operator to provide the RPC head block number to another value if desired. In Nitro, this will default to Finalized. Using Latest here makes sure that our tests remain fast without needing to change test setups across the whole codebase with options